src/components/elements/RegisterInput.tsx

```diff
@@ -0,0 +1,7 @@
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const RegisterInput = (props: InputProps) => {
+    return <input {...props} className="rounded-lg p-2 ml-2" required />
```

Comment:
Remove `ml-2`. Generally, we don't put padding or margin on just one side of a component. If needed, put the component inside a div then put `ml-2` in that div.